internal/rbd/nodeserver.go

```diff
@@ -870,3 +887,27 @@ func openEncryptedDevice(ctx context.Context, volOptions *rbdVolume, devicePath
 
 	return mapperFilePath, nil
 }
+
+// xfsSupportsReflink checks if mkfs.xfs supports the "-m reflink=0|1"
+// argument. In case it is supported, return true.
+func (ns *NodeServer) xfsSupportsReflink() bool {
+	// return cached value, if set
+	if xfsHasReflink != xfsReflinkUnset {
+		return xfsHasReflink == xfsReflinkSupport
+	}
+
```

Comment:
@nixpanic do we need this whitespace here?

Reply:
not required, but it splits two parts of unrelated code:
1. check the cached value
2. no cache, so test for it